code/style.css

```diff
@@ -1,4 +1,300 @@
+                                                            /***** BREAKPOINT INTO TABLET ****/
+                                                            
+                                                            .container-big-news {
+                                                                display: flex;
+                                                                justify-content: flex-start;
+                                                            }
+                                                            
+                                                            .container-smaller-news {
+                                                                display: flex;
+                                                                flex-direction: column;
+                                                            }
+                                                            
+                                                            @media (min-width: 768px) and (max-width: 991px) {
+                                                                .container-big-news {
+                                                                    width: 105%;
+                                                                    margin-left: -2%;
+                                                                }
+                                                                .container-smaller-news {
+                                                                    width: 105%;
+                                                                    flex-direction: row;
+                                                                    flex-wrap: wrap;
+                                                                    justify-content: space-between;
+                                                                }
+                                                                .news-card {
+                                                                    width: 42%;
+                                                                }
+                                                                .couture img {
+                                                                    margin-left: 5.5%;
+                                                                }
+                                                                .heading-couture {
+                                                                    margin-right: 30%;
+                                                                    padding-left: 6%;
+                                                                    padding-right: 8%;
+                                                                }
+                                                                .text-couture {
+                                                                    margin-right: -47.5%;
```

Comment:
Be careful with negative values as it makes the structure non-logical sometimes. Allocate space in the parent element instead (heading-couture).